pkg/models/docker.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package models
+
+/*
```

Comment:
what is this

Reply:
it's not used for now as we use docker cli to communicate with docker. 
this would proxy docker socket cli to talk to , commented out as it's currently unused